src/i2c.rs

```diff
@@ -1,16 +1,17 @@
-//! Inter-Integrated Circuit (I2C) bus
```

Comment:
What's the reason for the comment style change? C++ comments are not so typical in the Rust world.

Reply:
Oops, I think I started adding a larger doc chunk, but then changed my mind about what I wanted to do. I'll revert it until I get around to adding i2c docs

Reply:
Even then it should be possible to use Rust style comments. ;)

Reply:
Sure, but I kind of dislike that for large blocks, especially inserting new lines